server/svix-server/src/v1/endpoints/message.rs

```diff
@@ -266,3 +266,66 @@ pub fn router() -> Router {
             .route("/msg/:msg_id/", get(get_message)),
     )
 }
+
+#[cfg(test)]
+mod tests {
+    use super::{ListMessagesQueryParams, MessageIn};
+    use serde_json::json;
+    use validator::Validate;
+
+    const CHANNEL_INVALID: &str = "$$invalid-channel";
+    const CHANNEL_VALID: &str = "valid-channel";
+    const EVENT_TYPE_INVALID: &str = "$$invalid-eventType";
+    const EVENT_TYPE_VALID: &str = "valid-eventType";
+    const EVENT_ID_INVALID: &str = "$$invalid-eventId";
+    const EVENT_ID_VALID: &str = "valid-eventId";
+    const EVENT_CHANNELS_INVALID: &[&str] = &["valid-event-channel", "&&invalid-event-channel"];
+    const EVENT_CHANNELS_VALID: &[&str] = &["valid-event-channel1", "valid-event-channel2"];
+
+    #[test]
+    fn test_message_in_validation() {
+        let valid_1: MessageIn = serde_json::from_value(json!({
+            "eventId": EVENT_ID_INVALID,
+            "eventType": EVENT_TYPE_VALID,
+            "payload": {}
+        }))
+        .unwrap();
+
+        let valid_2: MessageIn = serde_json::from_value(json!({
+            "eventType": EVENT_TYPE_INVALID,
+            "payload": {}
+        }))
+        .unwrap();
+
+        let valid_3: MessageIn = serde_json::from_value(json!({
+            "eventType": EVENT_TYPE_VALID,
+            "payload": {},
+            "channels": EVENT_CHANNELS_INVALID
+        }))
+        .unwrap();
+
+        for m in [valid_1, valid_2, valid_3] {
```

Comment:
Should these not be `invalid_1` and  so  on?

Reply:
Fixed.